snowdrop-block/src/Snowdrop/Block/Demo.hs

```diff
@@ -0,0 +1,67 @@
+{-# LANGUAGE AllowAmbiguousTypes #-}
+
+module Snowdrop.Block.Demo
+       ( blockSync
+       ) where
+
+import           Universum
+
+import           Data.Default (Default) 
+
+import           Snowdrop.Block.StateConfiguration (BlkStateConfiguration (..))
+import           Snowdrop.Block.Configuration (BlkConfiguration (..), getCurrentBlockRef,
+                                               getPreviousBlockRef)
+import           Snowdrop.Block.Fork (ForkVerificationException (..))
+import           Snowdrop.Block.Types (BlockRef, CurrentBlockRef (..), RawBlund,
+                                       PrevBlockRef (..), RawBlk)
+import           Snowdrop.Core (ERwComp)
+import           Snowdrop.Util
+
+blockSync
+    :: forall blkType e id value ctx stateChgAccum blockChgAccum .
```

Comment:
`stateChgAccum` type parameter is not needed. Removing it will allow to delete `{-# LANGUAGE AllowAmbiguousTypes #-}`